src/commands/clear/index.ts

```diff
@@ -0,0 +1,105 @@
+import { Spinner } from "../../lib/index.js";
+import { SwankyCommand } from "../../lib/swankyCommand.js";
+import { ConfigError, FileError } from "../../lib/errors.js";
+import fs from "fs-extra"
+import path from "path"
+import { Args, Flags } from "@oclif/core";
+import { flagUsage } from "@oclif/core/lib/parser/help.js";
+
+type Folder = {name: string, contractName?: string, path: string}
+
+export default class Clear extends SwankyCommand<typeof Clear> {
+
+  static flags = {
+    all: Flags.boolean({
+      char: "a",
+      description: "Select all the project artifacts for delete"
+    }),
+  };
+
+  static args = {
+    contractName: Args.string({
+      name: "contractName",
+      required: false,
+      description: "Name of the contract artifact to clear",
+    }),
+  };
+
+  static async deleteFolder(path: string): Promise<void> {
```

Comment:
Does this need to be static? What's the benefit? If it's meant to be used by other classes, it should be a part of `lib`